content/helm-nginx/cleaningup.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Cleaning Up"
+date: 2018-08-07T08:30:11-07:00
+weight: 50
+---
+
+To remove all the objects that the Helm Chart created, we can use [Helm delete](https://docs.helm.sh/helm/#helm-delete).
+
+Before we delete it though, we can verify what we have running via the [Helm list](https://docs.helm.sh/helm/#helm-list) command:
+
+```
+helm list
+```
+
+You should see output similiar to below, which show that mywebserver is installed:
+
+```
+NAME            REVISION        UPDATED                         STATUS          CHART           APP VERSION     NAMESPACE
+mywebserver     1               Tue Nov 13 19:55:25 2018        DEPLOYED        nginx-1.1.2     1.14.1          default
+```
+
+It was a lot of fun; we had some great times sending HTTP back and forth, but now its time to delete this deployment.  To delete:
+
+```
+helm delete mywebserver
```

Comment:
don't you need to use '--purge' to delete helm chart completely?

Reply:
fixed